src/pudl/etl/cli.py

```diff
@@ -158,12 +148,18 @@ def pudl_etl(
         },
     }
 
+    # By default, limit CEMS and FERC 714 processing concurrency to prevent memory overload.
     tag_concurrency_limits = [
         {
             "key": "datasource",
             "value": "epacems",
-            "limit": epacems_workers,
-        }
+            "limit": 2,
+        },
+        {
+            "key": "datasource",
+            "value": "ferc714",
+            "limit": 1,
+        },
```

Comment:
As above, I don't think changed Dagster's configuration will have any impact on what happens inside the imputation function, which is doing a bunch of tensor math that's internally parallelized.